src/app/(home)/page.tsx

```diff
@@ -62,36 +105,63 @@ export default function HomePage() {
 
             {/* CTA Section */}
             <section>
-                <div className="relative z-10 mt-12 flex flex-col text-2xl font-black md:flex-row lg:mt-24 lg:text-3xl">
-                    <h3>New Members are</h3>
-                    <div className="mt-2 w-fit bg-purple px-2 md:ml-2 md:mt-0">
-                        <h3 className=" text-grey">Always Welcome</h3>
+                <div className="grid grid-cols-1 lg:grid-cols-2">
+                    <div
+                        className={`flex flex-col justify-around ${UPCOMING_EVENTS.length === 0 ? 'lg:col-span-2' : ''}`}
+                    >
+                        <div>
+                            <div className="relative z-10 mt-12 flex flex-col text-2xl font-black md:flex-row lg:mt-24 lg:text-3xl">
+                                <h3>New Members are</h3>
+                                <div className="mt-2 w-fit bg-purple px-2 md:ml-2 md:mt-0">
+                                    <h3 className=" text-grey">Always Welcome</h3>
+                                </div>
+                            </div>
```

Comment:
Make "New Members are" and "Always Welcome" go on seperate lines like this:

<img width="191" alt="image" src="https://github.com/user-attachments/assets/4cfbf49f-d70a-4a36-84e4-3e80b86a4b72">